tlr/stages-tlr/model-provider/src/main/java/edu/kit/kastel/mcse/ardoco/tlr/models/connectors/generators/code/java/JavaExtractor.java

```diff
@@ -80,9 +80,10 @@ private static ASTParser getJavaParser() {
         return parser;
     }
 
-    private static String[] getEntries(Path dir, String suffix) {
```

Comment:
Please use for example `predictFileTypesFromFolderRecursively` or `predictFileTypes`. This should also be more performant